sys/crypto/modes/ccm.c

```diff
@@ -129,24 +129,33 @@ int ccm_compute_adata_mac(cipher_t* cipher, uint8_t* auth_data,
 }
 
 
+/* Check if 'value' can be stored in 'num_bytes' */
+static inline int _fits_in_nbytes(size_t value, uint8_t num_bytes)
+{
+    /* Not allowed to shift more or equal that left operand width
+     * So we shift by maximun num bits of size_t -1 and compare to 1
```

Comment:
typos: `/that/than/` and `/maximun/maximum/`

Reply:
Thank you, fixed.